sweetercat/templates/detail.html

```diff
@@ -50,13 +50,17 @@ <h2>Planetary information</h2>
       <h2><a href="{{ planet['exolink'] }}" target="_blank">{{ planet['plName'] }}</a></h2>
       <h3>General info</h3>
       {% if planet['plMass']|string != '...' %}
-        <p>Mass: {{ planet['plMass'] }}M<sub>Jup</sub></p>
+        {% if planet['plMass'] < 0.314 %}
+          <p>Mass: {{ (planet['plMass']/0.00314)|round(3) }}M<sub>Earth</sub></p>
```

Comment:
Maybe be a bit more precise here: divide by 0.0031463518655061432 or multiply by 317.8284065946748
so it doesn't loose so much precision before the rounding.

got those values from
```python
import astropy.units as u
u.earthMass.to(u.jupiterMass)
u.jupiterMass.to(u.earthMass)
```